apps/web/tests/components/submission/components/Credential.test.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Credential } from '../../../../src/components/submission/components/Credential';
+import {
+  defaultSpecialtyValue,
+  registrationStatusOptions,
+  healthAuthorityOptions,
+  employmentOptions,
+} from '../../../../src/components/submission/validation/credential';
+import * as formik from 'formik';
+
+const mockSetFieldValue = jest.fn();
+Object.defineProperty(formik, 'useFormikContext', () => ({ setFieldValue: mockSetFieldValue }));
+
+jest.mock('next/router', () => ({
+  useRouter: () => ({
+    query: {
+      step: '3',
+    },
+  }),
+}));
+
+describe('Credential', () => {
+  it('should render', () => {
```

Comment:
I notice this has a ton of of checks in each it block. Conventional testing good practice suggests having only one test per block to make it easier to track down and fix issues. Whats your take?

Reply:
I think this comment should be on `should render the expected fields` but yeah I think you're right in general. In this case I am testing for the same thing just for different fields and it felt like it'd be too verbose to write a test for each field. So you're right and the question is "is this shortcut bad enough to fix"